src/urllib3/util/util.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional, Union
+
+
+def to_bytes(
```

Comment:
Since I wrote this code would be good to get another outside review (cc @pquentin @hodbn )